src/table/boot.rs

```diff
@@ -256,10 +295,13 @@ pub enum MemoryType {
 }
 
 /// A structure describing a region of memory.
+#[derive(Debug, Copy, Clone)]
 #[repr(C, packed)]
 pub struct MemoryDescriptor {
     /// Type of memory occupying this range.
     pub ty: MemoryType,
+    /// Skip 4 bytes as UEFI declares items in structs should be naturally aligned
+    pub padding: u32,
```

Comment:
Could leave the `padding` field in without `pub`, since this is sort-of an implementation detail.